### PR TITLE
Preload checks for blocking extensions

### PR DESCRIPTION
**Pull request by tharders**
Preload check for runtime now also allows hosted version and javascript module version.
Do we need to change the warning message now?

### CODE REVIEW

File: packages/linter/src/rules/BlockingExtensionsPreloaded.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { Context } from "../index";
+import { Rule } from "../rule";
+
+const blockingExtension = ["amp-experiment", "amp-dynamic-css-classes"];
```
Comment (sebastianbenz) [inline packages/linter/src/rules/BlockingExtensionsPreloaded.ts:4]: amp-story needs to go here as well.

File: packages/linter/src/rules/RuntimeIsPreloaded.ts [hunk 2]
```diff
@@ -3,14 +3,13 @@ import { Rule } from "../rule";
...
-    const attr = [
-      "href='https://cdn.ampproject.org/v0.js'",
-      "rel='preload'",
-      "as='script'",
-    ]
+    const jsAttr = ["href$='/v0.js'", "rel='preload'", "as='script'"]
       .map((s) => `[${s}]`)
       .join("");
-    const isPreloaded = $(`link${attr}`).length > 0;
+    const mjsAttr = ["href$='/v0.mjs'", "rel='modulepreload'"]
```
Comment (sebastianbenz) [inline packages/linter/src/rules/RuntimeIsPreloaded.ts:9]: We're currently not using `modulepreload` so please omit checking for this attribute

File: packages/linter/src/rules/BlockingExtensionsPreloaded.ts (hunk 1 above)
Comment (tharders) [reply packages/linter/src/rules/BlockingExtensionsPreloaded.ts]: done

File: packages/linter/src/rules/RuntimeIsPreloaded.ts (hunk 2 above)
Comment (tharders) [reply packages/linter/src/rules/RuntimeIsPreloaded.ts]: For mjs we now simply check if the rel attribute ends with preload, so normal preload (as it is used now) or modulepreload will be fine.

**Review by sebastianbenz — APPROVED** [review]

**Comment by sebastianbenz** [comment]
Thanks!

**Pull request merged**